src/pick.c

```diff
@@ -291,10 +292,10 @@ selected_choice(void)
 		case DEL:
 			if (cursor_position > 0) {
 				delete_between(
-				    query,
-				    query_length,
-				    cursor_position - 1,
-				    cursor_position);
+					       query,
+					       query_length,
+					       cursor_position - 1,
+					       cursor_position);
```

Comment:
Vim[1]. If your settings is the preferred style of pick lets keep it that way.

[1] https://github.com/mptre/dotfiles/blob/freebsd/vim/ftplugin/c.vim#L1

On Fri, Jan 15, 2016 at 07:28:55AM -0800, Calle Erlandsson wrote:
> > @@ -291,10 +292,10 @@ selected_choice(void)
> >  		case DEL:
> >  			if (cursor_position > 0) {
> >  				delete_between(
> > -				    query,
> > -				    query_length,
> > -				    cursor_position - 1,
> > -				    cursor_position);
> > +					       query,
> > +					       query_length,
> > +					       cursor_position - 1,
> > +					       cursor_position);
> 
> I think these instances were correctly indented. I use `cinoptions=:0,t0,+4,(4` in Vim. What editor and configuration do you use?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/thoughtbot/pick/pull/96/files#r49865276

-- 
:wq